tests/unittests/benchmark/task/profet/conftest.py

```diff
@@ -0,0 +1,119 @@
+import random
+from logging import getLogger as get_logger
+from pathlib import Path
+from typing import Dict, Tuple
+
+import numpy as np
+import pytest
+import torch
+from _pytest.fixtures import SubRequest
+from _pytest.monkeypatch import MonkeyPatch
+from _pytest.tmpdir import TempPathFactory
+from orion.benchmark.task.profet.profet_task import MetaModelConfig
+
+logger = get_logger(__name__)
+
+REAL_PROFET_DATA_DIR: Path = Path("profet_data")
+
+
+def seed_everything(seed: int) -> None:
+    random.seed(seed)
+    np.random.seed(seed)
+    torch.random.manual_seed(seed)
+    if torch.cuda.is_available():
+        torch.cuda.manual_seed_all(seed)
+
+
+def is_nonempty_dir(p: Path) -> bool:
+    return p.exists() and p.is_dir() and bool(list(p.iterdir()))
+
+
+@pytest.fixture(scope="session")
+def checkpoint_dir(tmp_path_factory):
+    return tmp_path_factory.mktemp("checkpoint_dir")
+
+
+@pytest.fixture(scope="session")
+def profet_input_dir(tmp_path_factory):
+    # Returns the `input_dir` to use for the Profet algorithm.
+    if REAL_PROFET_DATA_DIR.exists():
+        return REAL_PROFET_DATA_DIR
+    return tmp_path_factory.mktemp("profet_data")
+
+
+logger = get_logger(__name__)
+shapes: Dict[str, Tuple[Tuple[int, ...], Tuple[int, ...], Tuple[int, ...]]] = {
```

Comment:
Whenever we would add new subtasks we would need to come to the base config to add specific values. It would be preferable to use `self.shapes` and override it locally in subclasses instead. This way all relevant information specific to a subclass is contained locally.

Reply:
This comment is for shapes, y_min, y_max and c_min.

Reply:
Fixed in bc3e602c36a1910b5693673bc55e3ad19d27511d

Reply:
Are they values that we should hardcode in the public classes or rather sanity-check values for the tests? If the latter, I would be in favor of setting these values in the test classes instead.

Reply:
I think there is value in having these values be publicly displayed, it can give the users an idea of what the data loops like.